src/client/datascience/interactive-ipynb/nativeEditor.ts

```
@@ -688,4 +694,65 @@ export class NativeEditor extends InteractiveBase implements INotebookEditor {
             }
         }
     }
+
+    private async handleRunByLine(runByLine: IRunByLine) {
+        try {
+            // If there's any payload, it has the code and the id
+            if (runByLine.cell.id && runByLine.cell.data.cell_type !== 'messages') {
```

What if `cell_type=markdown or raw`.
Shouldn't we handle other cell types.

---

This should be impossible so it should just throw an exception unless it's a code cell